coldfront/overlays/ocp-staging/patches/deployments/coldfront-qcluster-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coldfront-qcluster-deployment
+spec:
+  selector:
+    matchLabels:
+      app: coldfront
+      coldfront-qcluster-deployment: coldfront-app
```

Comment:
Is this label required? It doesn't appear to be used anywhere else, and if we don't need it we can remove this entire `selector` section. Without the patched selector, we get the following from the base repository:

```
   selector:
     matchLabels:
       app: coldfront
       coldfront-deployment: coldfront-qcluster-app
```